setup/bin/copy_outputs.json

```diff
@@ -1,7 +1,7 @@
 {"data":{"bam": {"patterns": ["outputs/*.bam","outputs/*.bai"],"parallels": 5},
          "vcf": {"patterns": ["outputs/*.vcf","outputs/*.norm.vcf.gz","outputs/*.norm.vcf.gz.tbi","outputs/*.mutect.txt"],"parallels": 3},
          "maf": {"patterns": ["outputs/*.maf","outputs/*.portal.txt"],"parallels": 1},
-     "qc": {"patterns": ["outputs/consolidated_metrics_data/*","outputs/consolidated_metrics_data/images/*"],"parallels": 2},
+         "qc": {"patterns": ["outputs/*FP_base_counts.txt","outputs/*.asmetrics","outputs/*.hsmetrics","outputs/*.hstmetrics","outputs/*.ismetrics*","outputs/{}_CutAdaptStats.txt","outputs/{}_DiscordantHomAlleleFractions.txt","outputs/{}_FingerprintSummary.txt","outputs/{}_GcBiasMetrics.txt","outputs/{}_HsMetrics.txt","outputs/{}_InsertSizeMetrics_Histograms.txt","outputs/{}_MajorContamination.txt","outputs/{}_markDuplicatesMetrics.txt","outputs/{}_MinorContamination.txt","outputs/{}_post_recal_MeanQualityByCycle.txt","outputs/{}_pre_recal_MeanQualityByCycle.txt","outputs/{}_ProjectSummary.txt","outputs/{}_QC_Report.pdf","outputs/{}_SampleSummary.txt","outputs/{}_UnexpectedMatches.txt","outputs/{}_UnexpectedMismatches.txt","outputs/{}_concordance.pdf","outputs/{}_concordance.txt","outputs/{}_contamination.pdf","outputs/{}_contamination.txt","outputs/{}_cdna_contamination.txt"],"parallels": 2},
```

Comment:
We are actually no longer using copy_outputs.json. All workflow inputs and outputs are stored in bin/roslin_workflows.py. Copy_outputs.json is an artifact and should be removed. I will put those changes in.